### PR TITLE
fix(youtubeVideoMessage): use notice state to memoize formated message

### LINKED ISSUE #969: Click on a thumb make disappear video preview
**Describe the bug**
Following preview video feature https://github.com/dis-moi/extension/issues/888
if I click on thumb up, video preview displayed disappears. 

**To Reproduce**
1. Follow approfondir account
2. Go to https://opendecide.com/practices/225
3. click on detail
4. Scroll down and click on thumb up

**Desktop (please complete the following information):**
 - Windows Firefox

**Extension**
- Version proding 3.74.2
- Installed from  store

**Additional context**

### PR DESCRIPTION
**Pull request by gregoirelacoste**
closes #969

### CODE REVIEW

**Review by JalilArfaoui — COMMENTED** [review]
Can you please explain what was the problem ? What was happening ?

File: src/app/lmem/format/message.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import getMessageWithMediaPlayer from '../../utils/getMessageWithMediaPlayer';
 
 export const formatMessage = (message: string): string => {
-  return getMessageWithMediaPlayer(message) || message;
+  return getMessageWithMediaPlayer(message);
```
[the remark this answers is not in the document]
Comment (gregoirelacoste) [reply src/app/lmem/format/message.ts:4]: this function is now useless, you're right

**Comment by lutangar** [comment]
This need a rebase and a question is still unanswered.

**Comment by gregoirelacoste** [comment]
the issue : https://github.com/dis-moi/extension/issues/969
in summary : `getMessageWithMediaPlayer()` in render return null when we change tab a second time. So to avoid that I put `getMessageWithMediaPlayer` in component did mount and set state with the result

**Review by lutangar — CHANGES_REQUESTED** [review]
I first intended to accept this PR but a few thing bothering me...
`formatMessage` function should be kept, not sure I understand the memo issue, but if you add the message to the state, you only need to edit the `NoticeDetails.tsx`, all the other updates bring more confusion to the codebase imho.

**Comment by gregoirelacoste** [comment]
> `formatMessage`

this function do nothing anymore

**Pull request merged**